WebApiStart/Services/BookService.cs

```diff
@@ -0,0 +1,99 @@
+﻿using AutoMapper;
+using FluentValidation.Results;
+using NLog;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using WebApiStart.Database;
+using WebApiStart.Errors;
+
+using WebApiStart.Model;
+
+namespace WebApiStart.View
+{
+    public class BookService:IBookService
```

Comment:
use commenting where you are performing some actions this applies to all the logic that's being written
i.e validation, error handling, db implementation